common/models/templates/src/trie.ts

```diff
@@ -52,6 +52,37 @@ export interface Entry {
   weight: number;
 }
 
+/**
+ * Recursively sort the trie, in descending order of weight.
+ * @param node any node in the trie
+ */
+export function sortNode(node: Node, onlyLocal?: boolean) {
```

Comment:
The function needed to be moved to avoid a cyclic-import scenario; importing from trie-builder results in a counter-import attempt that was causing builds and tests to fail.